src/mono/mono/metadata/class-setup-vtable.c

```diff
@@ -1870,10 +1869,10 @@ mono_class_setup_vtable_general (MonoClass *klass, MonoMethod **overrides, int o
 
 				// If the slot is still empty, look in all the inherited virtual methods...
 				if ((vtable [im_slot] == NULL) && klass->parent != NULL) {
-					MonoClass *parent = klass->parent;
+					parent = klass->parent;
```

Comment:
This whole assignment is redundant, isn't it?  `parent` is already set to `klass->parent` when it's declared, isn't it?  Is `klass` ever reassigned?

Reply:
Doesn't look like it, so assignment could probably be eliminated, given that klass->parent remains the same during the loop.